mmdet/models/backbones/resnet.py

```diff
@@ -45,11 +52,11 @@ def forward(self, x):
         identity = x
 
         out = self.conv1(x)
-        out = self.bn1(out)
+        out = getattr(self, self.norm1)(out)
```

Comment:
I suggest adding some properties so that we can directly use `self.norm1(out)`.
```python
@property
def norm1(self):
    return getattr(self, self.norm1_name)
```

Reply:
We need to add 3 properties for bottleneck and 2 for basic block, is that ok?